app/warden/magic_link_token.rb

```diff
@@ -0,0 +1,15 @@
+class MagicLinkToken < Warden::Strategies::Base
+  def valid?
+    params[:token].present?
```

Comment:
This is quite a generic name for the param — could we qualify it? Maybe `magic_link_token` or sth?

Reply:
We've decided to leave this for something to refactor later.